roles/users/defaults/main.yml

```diff
@@ -1,5 +1,3 @@
 deploy_user: deploy
 deploy_group: deploy
 deploy_groups: www-data
```

Comment:
If `deploy_user` etc. are defined in `group_vars/all`, could you remove `roles/users/defaults/main.yml`?